docs/docs/how-it-works/02_infra-as-code.md

```diff
@@ -4,4 +4,28 @@ sidebar_position: 3
 title: infra-as-code
 ---
 
-This package is a legacy initiative to leverage pulumi in managing twilio configuration artifacts such as taskrouter entities. This package is a functional methodology and was setup to represent a typically vanilla flex account thats just been initialized. This package has not been maintained as the intention is to move towards a similar solution using the terraform provider, however what has been setup and used before is available for anyone looking to use it in the interim. More details on how to use this can be found [here](https://www.twilio.com/blog/intro-to-infrastructure-as-code-with-twilio-part-1). Currently there are no dependencies in this package that need to be used and is here only for reference. Feel free to remove this if not utilized.
+This package contains the alternative infastructure as code solutions, right now only one is in use terraform
```

Comment:
Not sure I would call it alternative if we are using terraform as part of the happy-path pipeline scripts!